apps/SageAccountWeb/src/App.tsx

```diff
@@ -28,70 +24,65 @@ import { getSearchParam } from 'URLUtils'
 import './App.scss'
 import AppInitializer from './AppInitializer'
 import LoginPage from './LoginPage'
-import generalTheme from './style/theme'
 import { QueryClient, QueryClientProvider } from 'react-query'
+import ApplicationSessionManager from 'synapse-react-client/dist/utils/apputils/session/ApplicationSessionManager'
 
 const isCodeSearchParam = getSearchParam('code') !== undefined
 const isProviderSearchParam = getSearchParam('provider') !== undefined
 const isInSSOFlow = isCodeSearchParam && isProviderSearchParam
 
-// theme is a merge of a general theme and particular color pallettesfor the source app
-const theme = createTheme(generalTheme)
```

Comment:
Theme will be created (once only!) in the AppInitializer, since we also fetch the app colors there.